src/Symfony/Component/Serializer/Encoder/XmlEncoder.php

```diff
@@ -224,10 +230,20 @@ final protected function isElementNameValid(string $name): bool
      */
     private function parseXml(\DOMNode $node, array $context = []): array|string
     {
+        $nodeName = $node->nodeName;
+
         $data = $this->parseXmlAttributes($node, $context);
 
         $value = $this->parseXmlValue($node, $context);
 
+        if (\is_array($value)
+            && ($childNodeName = $node->firstChild?->nodeName)
+            && 1 === \count($value)
+            && \in_array($nodeName, $context[self::FORCE_COLLECTION] ?? $this->defaultContext[self::FORCE_COLLECTION], true)
```

Comment:
I think it could be great to accept the value `*` as an "every tag" value, WDYT?

Reply:
Maybe, however I'm still unsure how this PR would be welcome. I'm okay to continue working on it, but I'd like to get a green light before investing too much time on it 😅 

But if it get a positive feedback (which doesn't seem to be the case for the moment) I can consider adding a test case for this, maybe using xpath also 🤔